boxfile.yml

```diff
@@ -131,6 +144,10 @@ worker.sidekiq:
       schedule: '30 00 * * *'
       command: 'bundle exec rake mastodon:push:clear'
 
+    - id: update_remote_avatars
```

Comment:
Remove this one

Reply:
Is there already a mechanism for relaying avatar changes through the Fediverse as hey happen, then? I'd like to show my users the current avatar of anyone they follow, and this _appears_ to be the only way to make it happen, so I'd want to comment it out rather than remove it entirely if there's no alternative already in place.

Reply:
Yes, obviously. This cron is troubleshooting.

Reply:
Where does it state that, exactly? This is the only place I've seen that mentioned...

Reply:
I don't see `update_remote_avatars` listed in that doc at all.